build/bundle.go

```diff
@@ -16,6 +16,13 @@ type BundleSpec struct {
 }
 
 type Bundle struct {
+	// actors version in this bundle
 	Version actors.Version
+	// release id
 	Release string
+	// bundle path: optional, uses the appropriate release bundle if unset
+	Path string
+	// development version; when set, in conjunction with path, it will always
+	// load the bundle to the blockstore
+	Development bool
 }
```

Comment:
nit: use the Godocs style for these comments.

Reply:
ok

Reply:
done.